### PR TITLE
SSI Guide: Revoke Credential

### PR DESCRIPTION
**Pull request by bobbilee19**
Here is version one, tested, and split into two guides. Would love to hear your thoughts @decentralgabe.

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]
looks great!

File: site/docs/ssi/revoke-credentials.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+sidebar_position: 7
+title: Revoke Credentials
+hide_title: true
+---
+
+# Revoke a Credential
+
+By now you should be familiar with creating a credential. Notably, that upon forming a request to [create a credential](/docs/apis/ssi-service#tag/Credentials/paths/~1v1~1credentials/put) there are a number of possible request values, two of which are `revocable` and `suspendable`.
+
+These options are exposed to give issuers the ability to specify [status for credentials](credential-status) they create in the service. In this guide we'll go over how to revoke a credential!
```
Comment (angiejones) [inline site/docs/ssi/revoke-credentials.md:11]: Move first sentence to previous paragraph, and change exclamation point to a period in the second sentence.

File: site/docs/ssi/revoke-credentials.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+## Create a Revocable Credential
```
Comment (angiejones) [inline site/docs/ssi/revoke-credentials.md:13]: Let's move this entire section to the Credential Status guide since the purpose of this guide is simply to revoke an existing credential, and the other guide is purposed to teach devs how to create credentials that include status.

I think all that's needed in this guide is what you have under step 4

File: site/docs/ssi/revoke-credentials.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  "suspended": false
+  //highlight-end
+}
+```
+
+### 4. Revoke the Credential
+
+As the creator of the credential we're able to change the status of the credential we've created.
+
+To do so, we would make request to the [update credential status endpoint](/docs/apis/ssi-service#tag/Credentials/paths/~1v1~1credentials~1%7Bid%7D~1status/put) which is a `PUT` request to `/v1/credentials/{id}/status`. At present, the endpoint accepts boolean values for the status(es) the credential supports.
```
Comment (angiejones) [inline site/docs/ssi/revoke-credentials.md:154]: > As the creator of a credential, you're able to change the status of that credential.
> 
> To do so, make a `PUT` request to `/v1/credentials/{id}/status` and specify a boolean value for the status you'd like to change.
> 
> Given a credential was created with `revocable` as true, you can revoke that credential by setting `revoked` to true:
> 
> ```bash
> curl -X PUT localhost:8080/v1/credentials/8f9d58b2-c978-4317-96bd-35949ce76121/status -d '{ "revoked": true }'
> ```
> 
> Upon success, you'll see the following response:
> 
> ```json
> { 
>   "revoked": true
> }
> ```
> 
> _Note that if the credential is also suspendable, the `suspended` status will also appear in the response above._
>

File: site/docs/ssi/revoke-credentials.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+Let's update the credential's status to revoked:
```
Comment (angiejones) [inline site/docs/ssi/revoke-credentials.md:156]: can remove lines 156-174

File: site/docs/ssi/revoke-credentials.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  "revoked": true,
+  "suspended": false
+}
+```
+
+Making a request as we did in step 3 should now show the same response.
+
+The credential is now **revoked!**
+
+:::note
```
Comment (angiejones) [inline site/docs/ssi/revoke-credentials.md:176]: > :::note
> It is possible to reverse the status of a credential. To do so, make a `PUT` request to `/v1/credentials/{id}/status`, and set the value of `revoked` to false.
> :::

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/credential-status.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+Then, for each new credential an issuer creates for a given schema:
+- A new credential status credential is created or an existing credential status credential is used.
+- The credentials contain a reference to the status list credential contained in the credential's `credentialStatus` property, which can be used by verifiers to check the status of the credential.
+
+#### Bitstring and Herd Privacy:
+Status credentials use a bitstring which can provide herd privacy for credential holders — in simpler terms this means that many credentials can be represented in a single bitstring, so it is not clear which credential/holder a verifier is requesting information about — this is great for holder privacy!
+</details>
+
+## Example Credential Status Credential:
```
[the remark this answers is not in the document]
Comment (angiejones) [reply site/docs/ssi/credential-status.md:35]: or ## "Credential Status" Credential

The quotes will help with readability

[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/credential-status.md:35]: that's what i thought at first, but if you keep reading the other credential we make as a SLC as well in it.

File: site/docs/ssi/credential-status.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+In this example below we have a **status list credential** which is used for many credentials issued by the issuer identified by the DID `did:example:12345`:
```
[the remark this answers is not in the document]
Comment (angiejones) [reply site/docs/ssi/credential-status.md:36]: ah ok cool. @bobbilee19 we can go with this

> Here is an example of a status list credential. Note that in addition to being of type `VerifiableCredential`, it also is of type `StatusList2021Credential`. Also note the `credentialSubject.type` property is set to `Status2021`:

[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/credential-status.md:36]: Let me know what you like better.

File: site/docs/ssi/revoke-credentials.md (hunk 3 above)
Comment (bobbilee19) [reply site/docs/ssi/revoke-credentials.md:154]: This is what the response is even though I didn't set suspendable so going to leave as is unless you got something different?...

```json
{ 
  //highlight-next-line
  "revoked": true,
  "suspended": false
}

File: site/docs/ssi/credential-status.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/credential-status.md]: I put in yours for now.

File: site/docs/ssi/revoke-credentials.md (hunk 3 above)
Comment (angiejones) [reply site/docs/ssi/revoke-credentials.md]: @decentralgabe why is the suspendable status here even though only revocable was specified when creating the credential?

**Review by angiejones — APPROVED** [review]
